#request
string training_set_path
---
#response
bool success
